Formula/ddcctl.rb

```diff
@@ -0,0 +1,19 @@
+class Ddcctl < Formula
+  desc "DDC monitor controls (brightness) for Mac OSX command-line"
+  homepage "https://github.com/kfix/ddcctl"
+  url "https://github.com/kfix/ddcctl/archive/refs/tags/v0.tar.gz"
+  sha256 "8440f494b3c354d356213698dd113003245acdf667ed3902b0d173070a1a9d1f"
+  license :public_domain
+
+  depends_on :macos
+
+  def install
+    bin.mkpath
+    system "make", "install", "INSTALL_DIR=#{bin}"
+  end
+
+  test do
+    output = shell_output("#{bin}/ddcctl -d 10 -b 100", Hardware::CPU.arm? ? 255 : 1)
```

Comment:
```suggestion
    output = shell_output("#{bin}/ddcctl -d 10 -b 100", 1)
```
That's odd. I could swear it said it expected `255` last time. Now it says it expected `1`?

Maybe I misread. Sorry!

Reply:
Might also be because I changed display number from 1 to 10. I just changed it to 100 so now it should output the same return code and error consistently as long as you have less than 100 monitors attached to your machine which I think is a fair assumption.